djoser/conf.py

```diff
@@ -38,6 +39,7 @@
     },
     'LOGOUT_ON_PASSWORD_CHANGE': False,
     'USER_EMAIL_FIELD_NAME': 'email',
+    'REREGISTRATION_SHOW_RESPONSE': True,
```

Comment:
I had it mixed up in what I wrote for the docs : the **Default** was set to True, but in-line the description was False. Fixing the docs to match.